guides/common/modules/proc_configuring-smart-proxy-server-with-custom-ssl-certificates-to-generate-and-sign-puppet-certificates.adoc

```diff
@@ -56,17 +56,31 @@ Retain a copy of the example `{foreman-installer}` command from the output for i
 --puppet-server true \
 --puppet-server-ca "true"
 ----
-. On {SmartProxyServer}, generate Puppet certificates for all other {SmartProxies} that you configure for load balancing, except this first system where you configure Puppet certificates signing:
+. On {SmartProxyServer} that is the Puppetserver Certificate Authority, stop the Puppet server:
 +
-[options="nowrap", subs="+quotes,attributes"]
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# systemctl stop puppetserver
+----
+. Generate Puppet certificates for all other {SmartProxyServers} that you configure for load balancing, except the system where you first configured Puppet certificate signing:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# puppet cert generate _{smartproxy-example-com}_ \
---dns_alt_names=_{loadbalancer-example-com}_
+# puppetserver ca generate \
+--ca-client \
+--certname _{smartproxy-example-com}_ \
+--subject-alt-names _{loadbalancer-example-com}_
 ----
 +
-This command creates the following files on the Puppet certificate signing {SmartProxyServer} instance:
+This command creates the following files:
 +
-* `/etc/puppetlabs/puppet/ssl/certs/ca.pem`
-* `/etc/puppetlabs/puppet/ssl/certs/{smartproxy-example-com}.pem`
-* `/etc/puppetlabs/puppet/ssl/private_keys/{smartproxy-example-com}.pem`
-* `/etc/puppetlabs/puppet/ssl/public_keys/{smartproxy-example-com}.pem`
+* `/etc/puppetlabs/puppet/ssl/certs/_{smartproxy-example-com}_.pem`
+* `/etc/puppetlabs/puppet/ssl/private_keys/_{smartproxy-example-com}_.pem`
+* `/etc/puppetlabs/puppet/ssl/public_keys/_{smartproxy-example-com}_.pem`
+* `/etc/puppetlabs/puppetserver/ca/signed/_{smartproxy-example-com}_.pem`
+. Restart the Puppet server:
```

Comment:
Nit: technically I think you're correct, but my mind using `restart` here and `start` in the command was inconsistent. I'll leave this up to the technical writers, but wonder if it should be
```suggestion
. Start the Puppet server again:
```